File: U1/EXAMEN/BanquitoServer_Soap_DotNet_GR01/BanquitoServer_Soap_DotNet_GR01/DTOs/CuotaAmortizacionDTO.cs
using System.Runtime.Serialization;

namespace BanquitoServer_Soap_DotNet_GR01.DTOs
{
    /// <summary>
    /// DTO para cuota de amortización
    /// </summary>
    [DataContract]
    public class CuotaAmortizacionDTO
    {
        [DataMember]
        public int NumeroCuota { get; set; }

        [DataMember]
        public decimal ValorCuota { get; set; }

        [DataMember]
        public decimal Interes { get; set; }

        [DataMember]
        public decimal CapitalPagado { get; set; }

        [DataMember]
        public decimal Saldo { get; set; }
    }
}

File: U2/PROYECTO/04. SERVIDOR/BanquitoServer_Restfull_Java_GR01/src/main/java/ec/edu/monster/DTOs/CuotaAmortizacionDTO.java
package ec.edu.monster.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;

/**
 * DTO para cuota de amortización
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CuotaAmortizacionDTO {
    private Integer numeroCuota;
    private BigDecimal valorCuota;
    private BigDecimal interes;
    private BigDecimal capitalPagado;
    private BigDecimal saldo;
}
